src/js/search/widgets/infinite_scroll.js

```diff
@@ -19,18 +19,24 @@ function renderTemplate(template, res) {
   return results;
 }
 
-function scrolledNearBottom(el, offset) {
-  return (el.scrollHeight - el.scrollTop) < offset;
+function scrolledNearBottom(scrollTop, offset) {
+  var bodyHeight = document.querySelector('body').clientHeight;
+  var pageHeight = document.documentElement.clientHeight;
+  return (bodyHeight - scrollTop - pageHeight) < offset;
```

Comment:
@mikaa123 Yeah, I think `contentHeight` and `containerHeight` make the most sense--great idea!